src/services/auth/service/developer-resource/projects.ts

```diff
@@ -0,0 +1,153 @@
+    };
+
+type ProjectIdentity = Identity & { identity_provider: string; identity_type: string };
+
+/**
+ * @see https://docs.globus.org/api/auth/reference/#projects_resource
+ */
+export type Project = {
+  id: string;
+  project_name: string;
+  display_name: string;
+  contact_email: string;
+  admins: {
+    identities: ProjectIdentity[];
+    groups: components['schemas']['GroupReadModel'][];
+  } & ProjectAdminIDs;
+};
+
+/**
+ * Return a specific project by id if the authenticated entity is an admin of that project.
+ * @see https://docs.globus.org/api/auth/reference/#get_projects
+ */
+export const get = function (project_id, options = {}, sdkOptions?) {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: SCOPES.MANAGE_PROJECTS,
+      path: `/v2/api/projects/${project_id}`,
+      method: HTTP_METHODS.GET,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<string, Record<string, any>>;
+
+/**
+ * Return a list of projects the authenticated entity is an admin of.
+ * @see https://docs.globus.org/api/auth/reference/#get_projects
+ */
+export const getAll = function (options = {}, sdkOptions?) {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: SCOPES.MANAGE_PROJECTS,
+      path: `/v2/api/projects`,
+      method: HTTP_METHODS.GET,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethod<{
+  headers?: Record<string, string>;
+  payload?: never;
+}>;
+
+/** Note that project_name is an alias for display_name and is not submitted on create or update */
+type ProjectCreate = Pick<Project, 'display_name' | 'contact_email'> & ProjectAdminIDs;
+
+/**
+ * Create a new project with a set of admins
+ *
+ * @see https://docs.globus.org/api/auth/reference/#create_project
+ */
+export const create = function (options, sdkOptions?): Promise<JSONFetchResponse<Project>> {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: SCOPES.MANAGE_PROJECTS,
+      path: `/v2/api/projects`,
+      method: HTTP_METHODS.POST,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethod<{
+  query?: never;
+  payload: ProjectCreate;
+}>;
+
+/**
+ * Update a project by id. All fields are optional.
+ *
+ * @see https://docs.globus.org/api/auth/reference/#update_project
+ */
+export const update = function (
+  project_id,
+  options,
+  sdkOptions?,
+): Promise<JSONFetchResponse<Project>> {
+  return serviceRequest(
+    {
+      service: ID,
+      scope: SCOPES.MANAGE_PROJECTS,
+      path: `/v2/api/projects/${project_id}`,
+      method: HTTP_METHODS.PUT,
+    },
+    options,
+    sdkOptions,
+  );
+} satisfies ServiceMethodDynamicSegments<
+  string,
+  {
+    query?: never;
+    payload: Partial<ProjectCreate>;
+  }
+>;
+
+/**
+ * Delete a project by id.
+ *
+ * @see https://docs.globus.org/api/auth/reference/#delete_project
+ */
+export const remove = function (
+  project_id,
+  options?,
+  sdkOptions?,
+): Promise<JSONFetchResponse<Project>> {
```

Comment:
Similar note here on the delete, I imagine the delete response isn't the record?